test/test_create_entities.py

```diff
@@ -0,0 +1,135 @@
+# limitations under the License.
+import unittest
+
+from test.unified_format import UnifiedSpecTestMixinV1
+
+
+class TestCreateEntities(unittest.TestCase):
+
+    def setUp(self):
+        self.scenario_runner = UnifiedSpecTestMixinV1()
+
+    def test_store_events_as_entities(self):
+        spec = {
+            "description": "blank",
+            "schemaVersion": "1.2",
+            "createEntities": [
+                {
+                    "client": {
+                        "id": "client0",
+                              "storeEventsAsEntities": [
+                                  {
+                                      "id": "events1",
+                                      "events": [
+                                          "PoolCreatedEvent",
+                                      ]
+                                  }
+                              ]
+                    }
+                },
+            ],
+            "tests": [
+                {
+                    "description": "foo",
+                    "operations": []
+                }
+            ]
+        }
+        self.scenario_runner.TEST_SPEC = spec
+        self.scenario_runner.setUp()
+        self.scenario_runner.run_scenario(spec["tests"][0])
+        final_entity_map = self.scenario_runner.entity_map
+        self.assertIn("events1", final_entity_map)
+        self.assertGreater(len(final_entity_map["events1"]), 0)
+
+    def test_store_all_others_as_entities(self):
+        spec = {
+            "description": "Find",
+            "schemaVersion": "1.2",
+            "createEntities": [
+                {
+                    "client": {
+                        "id": "client0",
+                        "uriOptions": {
+                            "retryReads": True
+                        },
+                    }
+                },
+                {
+                    "database": {
+                        "id": "database0",
+                        "client": "client0",
+                        "databaseName": "dat"
+                    }
+                },
+                {
+                    "collection": {
+                        "id": "collection0",
+                        "database": "database0",
+                        "collectionName": "dat"
+                    }
+                }
+            ],
+
+            "tests": [
+                {
+                    "description": "test loops",
+                    "operations": [
+                        {
+                            "name": "loop",
+                            "object": "testRunner",
+                            "arguments": {
+                                "storeIterationsAsEntity": "iterations",
+                                "storeSuccessesAsEntity": "successes",
+                                "storeFailuresAsEntity": "failures",
+                                "storeErrorsAsEntity": "errors",
+                                "numIterations": 100,
```

Comment:
I like numIterations idea! Let's reduce this to 5 to speed up the test.

Reply:
Done.